### PR TITLE
INT-5172 add overview.md

### PR DESCRIPTION
**Pull request by adrianpowell**
Adds an OVERVIEW.md file to flesh out some of the technical documentation within the project

Jira: https://issues.sonatype.org/browse/INT-5172

### CODE REVIEW

File: OVERVIEW.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Document Purpose
+
+This page provides high-level technical information regarding the **IBM Operator for Sonatype Lifecycle (IQ)**
+
+
+
```
Comment (collinpeters) [inline OVERVIEW.md:6]: Triple newline

File: OVERVIEW.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* Red Hat build service
+* Red Hat CodeReady Containers (CRC)
+
+## Data Persistence
+
+N/A
+
+## Local Development
+
+See the [REAME.md](README.md) file
```
Comment (collinpeters) [inline OVERVIEW.md:31]:
```suggestion
See the [README.md](README.md) file
```

**Review by collinpeters — APPROVED** [review]
+1 some minor comments including a typo

File: OVERVIEW.md (hunk 1 above)
Comment (adrianpowell) [reply OVERVIEW.md]: > 

updated

**Review by fernau — APPROVED** [review]
LGTM

**Pull request merged**